app/views/shared/export/_plan.erb

```diff
@@ -43,7 +43,7 @@
             <% blank = answer.present? ? answer.text.gsub(/<\/?p>/, '').gsub(/<br\s?\/?>/, '').chomp.blank? : true %>
```

Comment:
using answer.is_valid? would help to simplify this context. we should address this later.

Reply:
ah yeah. adding the code to ignore the html to this is_valid would be better.
Could probably make use of the Nokogiri gem to load the answer.text value into html and then do a check on that html object's `.text()` to make sure its ignoring all markup when checking the answer